app/infrastructure/middlewares/performance.py

```
@@ -1,44 +1,75 @@
+        if request.url.path in self.ignored_paths:
+            return await call_next(request)
+
         start_time = time.perf_counter()
         exception_occurred = False
-        status_code: Optional[int] = None
+        status_code: int = 500
 
         try:
             response = await call_next(request)
             status_code = response.status_code
             return response
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)